README.md

```diff
@@ -98,8 +98,15 @@ from the up-to-date code should fix such issues.
 
 By default, ansible will prompt for API token and other variables. To fully
 automate the deployment, you can pass the variables as arguments to the
-`ansible-playbook` command with `-e key=value`. To find the variable names,
-check the ansible playbooks.
+`ansible-playbook` command with `-e key=value`. Arguments might be seen in
+the following table. Availability of arguments is also mentioned in the table.
+
+| Key                  | Value              | Production | Development |
+| -------------------- |------------------- | :--------: | :---------: |
+| github_token         | GitHub API token   | x          | x           |
```

Comment:
I'll probably win a *Nitpick of the Year* award for this, but... 

Highlighting the `keywords` would look much more awesome.

Reply:
Yes, the Nitpick of the Year award is definitely yours. :+1: :laughing: 

But, how would you imagine to highlight them? Is enough to make them **bold** or is _italic_ better choice?

Reply:
Please `highlight` with backticks